source/collection-management/tests/clustered-indexes.yml

```diff
@@ -0,0 +1,102 @@
+description: "clustered-indexes"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "5.3"
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name ts-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name test
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents: []
+
+tests:
+  - description: "createCollection with clusteredIndex"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+          clusteredIndex:
+            key: { _id: 1 }
+            unique: true
+            name: &index0name "test index"
+      - name: assertCollectionExists
+        object: testRunner
+        arguments:
+          databaseName: *database0Name
+          collectionName: *collection0Name
+
+  - description: "listCollections includes clusteredIndex"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+          clusteredIndex:
+            key: { _id: 1 }
+            unique: true
+            name: &index0name "test index"
+      - name: listCollections
+        object: *database0
+        arguments:
+          filter: { name: { $eq: *collection0Name } }
+        expectResult: [
+          {
+            name: *collection0Name,
+            options: {
+              clusteredIndex: {
+                key: { _id: 1 },
+                unique: true,
+                name: *index0name,
+                v: 2,
```

Comment:
Independent of my suggested change below, should we bother asserting that `v: 2` is set in the result? This seems like an assumption on the server-side, which could possibly change in the future. I assume `unique: true` is closely related to the design of this feature, but I think we can relax the `v` assertion (or perhaps just expect `{ $$type: [ int, long ] }`).

Reply:
Good call, the actual value here isn't relevant - relaxed to just assert on type.